tests/product/test_control.py

```diff
@@ -65,37 +65,10 @@ def test_server_start_one_host_started(self):
         self.assert_start_with_one_host_started(
             self.cluster.internal_slaves[0])
 
-    def test_server_start_all_started(self):
-        self.setup_cluster(NoHadoopBareImageProvider(), STANDALONE_PRESTO_CLUSTER)
-        start_output = self.run_prestoadmin('server start').splitlines()
-        process_per_host = self.get_process_per_host(start_output)
-        self.assert_started(process_per_host)
-
-        start_output = self.run_prestoadmin(
-            'server start', raise_error=False).splitlines()
-        self.assertRegexpMatchesLineByLine(
-            start_output,
-            self.expected_port_error(self.cluster.all_internal_hosts())
-        )
-
-    def test_server_stop_not_started(self):
```

Comment:
This can be removed because `test_server_restart_nothing_started()` covers this scenario (the first step of restart is a stop).